src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -375,6 +376,23 @@ public PaginatedResult getProjects(final Tag tag) {
         return getProjects(tag, false, false, false);
     }
 
+    /**
+     * Returns an optional containing the project version with the latest BOM import date.
+     * @param name the name of the project
+     * @return the version of the project with the latest BOM update date
+     */
+    @Override
+    public Optional getLastImportedVersionProject(final String name) {
```

Comment:
`Optional` accepts a type argument, so this should be:

```suggestion
    public Optional<Project> getLastImportedVersionProject(final String name) {
```

Reply:
Thanks. I wasn't entirely happy with my solution, especially with the need to cast. Yours is more elegant.